packages/core/src/components/DataTable.story.tsx

```diff
@@ -144,8 +166,10 @@ export class SearchDemo extends React.Component {
           </Spacing>
           <div style={{ flexGrow: 1 }}>
             <DataTable
+              renderers={renderers}
               autoHeight
```

Comment:
I wonder if it'd be worth changing `autoHeight` to `autoTableHeight` to distinguish this from row height 🤔

Reply:
renaming it would be a breaking change